api/src/api.ts

```diff
@@ -154,5 +154,27 @@ export const plugin = {
         },
       },
     });
+
+    server.route({
+      method: "GET",
+      path: "papers/arxiv:{arxivId}/version",
+      handler: async (request, h) => {
+        const arxivId = request.params.arxivId;
+        const version = await dbConnection.getLatestProcessedArxivVersion({ arxiv_id: arxivId });
+        if (version < 0) {
+          // We don't have version info for this ID
+          return h.response().code(404);
+        }
+        // TODO (mjlangan): solidify the response structure (maybe 'maxVersion' or 'latestVersion'?)
+        return h.response({ version }).code(200);
```

Comment:
@mjlangan is this open question something we should discuss as part of this PR?

Reply:
We can but until this is actually being used, it's not important. Maybe `version` is plenty fine for now.

Reply:
Sounds good to me! Should we leave in the TODO note or take it out?

Reply:
I'll remove it.